docs/Users/Tutorials/External_Bears.rst

```diff
@@ -0,0 +1,511 @@
+External Bears
+==============
+
+Welcome. This tutorial hopes to teach you how to use the
+``@external_bear_wrap`` decorator in order to write Bears in languages other
+than python.
+
+.. note::
+
+  This tutorial assumes that you already know the basics. If you are new please
+  refer to
+  :doc:`this link<Writing_Bears>`.
+
+  If you are planning to create a bear that uses an already existing tool (aka
+  linter), please refer to
+  :doc:`this link instead<Linter_Bears>`.
+
+Why is This Useful?
```

Comment:
Is this relevant to this tutorial? People looking to write a bear already know what coala is.